[PROD] add metrics in docs page, fix warning formattingissue (#5689)
DataDog/integrations-core: openshift/README.md

| Live Container monitoring      | ❌                                       | ❌                                           | ✅                                             |
| Live Process monitoring        | ❌                                       | ❌                                           | ✅                                             |

<div class="alert alert-warning">

> **OpenShift 4.0+**: If you used the OpenShift installer on a supported cloud provider, you will need to deploy the Agent with `hostNetwork: true` to get host tags/aliases as access to metadata servers from PODs network is otherwise restricited.

Reviewer: ```suggestion
**OpenShift 4.0+**: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with `hostNetwork: true` in the `datadog.yaml` configuration file to get host tags/aliases as access to metadata servers from PODs network, which otherwise is restricted.
```

> <bold>OpenShift 4.0+</bold>: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with <code>hostNetwork: true</code> in the <code>datadog.yaml</code> configuration file to get host tags/aliases as access to metadata servers from PODs network, which otherwise is restricted.

Reviewer 2: ```suggestion
<bold>OpenShift 4.0+</bold>: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with <code>hostNetwork: true</code> in the <code>datadog.yaml</code> configuration file to get host tags and aliases. Access to metadata servers from the PODs network is otherwise restricted.
```

- `volumes: hostPath`: Accesses the Docker socket and the host's `proc` and `cgroup` folders, for metric collection.
- `SELinux type: spc_t`: Accesses the Docker socket and all processes' `proc` and `cgroup` folders, for metric collection. You can read more about this type [in this Red Hat article][7].

<div class="alert alert-info">

> Do not forget to add <a href="https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/?tab=k8sfile#configure-rbac-permissions">datadog-agent service account</a> to the newly created <a href="https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/manifests/openshift/scc.yaml">datadog-agent SCC</a> by adding <code>system:serviceaccount:<datadog-agent namespace>:<datadog-agent service account name></code> to the <code>users</code> section.

Reviewer 2: ```suggestion
Do not forget to add a <a href="https://docs.datadoghq.com/agent/kubernetes/daemonset_setup/?tab=k8sfile#configure-rbac-permissions">datadog-agent service account</a> to the newly created <a href="https://github.com/DataDog/datadog-agent/blob/master/Dockerfiles/manifests/openshift/scc.yaml">datadog-agent SCC</a> by adding <code>system:serviceaccount:<datadog-agent namespace>:<datadog-agent service account name></code> to the <code>users</code> section.
```

</div>

<div class="alert alert-warning">

> <b>OpenShift 4.0+</b>: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with <code>allowHostNetwork: true</code> in the <code>datadog.yaml</code> configuration file to get host tags/aliases as access to metadata servers from PODs network, which otherwise is restricted.

Reviewer 2: ```suggestion
<b>OpenShift 4.0+</b>: If you used the OpenShift installer on a supported cloud provider, you must deploy the Agent with <code>allowHostNetwork: true</code> in the <code>datadog.yaml</code> configuration file to get host tags and aliases. Access to metadata servers from the PODs network is otherwise restricted.
```

### Metrics

> See [metadata.csv][10] for a full list of provided metrics by this integration.

Reviewer 3: ```suggestion
See [metadata.csv][10] for a list of metrics provided by this integration.
```
